pkg/apic/auth/apicauth.go

```diff
@@ -529,10 +529,10 @@ func (ctg *channelTokenGetter) Close() error {
 	return nil
 }
 
-// TokenAuth implements the PerRPCCredentials interface
+// TokenAuth -
 type TokenAuth struct {
-	tenantID       string
-	tokenRequester PlatformTokenGetter
+	TenantID       string
+	TokenRequester TokenGetter
```

Comment:
I may have missed it, but did not see why this is not public

Reply:
I left these public because the `NewTokenAuth` method sets the `TokenRequester` field and I want to be able to use my own. In this case It's the `TokenGetter` on the agent.

Reply:
Makes sense, thank you sir